lib/spack/spack/test/cmd/view.py

```diff
@@ -33,10 +34,11 @@ def test_view_link_type(
     view(cmd, viewpath, 'libdwarf')
     package_prefix = os.path.join(viewpath, 'libdwarf')
     assert os.path.exists(package_prefix)
-    assert os.path.islink(package_prefix) == (not cmd.startswith('hard'))
+    assert os.path.islink(package_prefix) == (cmd in ('symlink', 'add'))
 
 
-@pytest.mark.parametrize('cmd', ['hardlink', 'symlink', 'hard', 'add'])
+@pytest.mark.parametrize('cmd', ['hardlink', 'symlink', 'hard', 'add',
+                                 'copy', 'relocate'])
```

Comment:
Same here

Reply:
I used a different method that I felt was cleaner to make this clearer, let me know if that works for you.